packages/core/src/oidc/init.ts

```diff
@@ -36,11 +36,13 @@ export default async function initOidc(app: Koa): Promise<Provider> {
       keys,
     },
     features: {
+      userinfo: { enabled: true },
       revocation: { enabled: true },
       introspection: { enabled: true },
       devInteractions: { enabled: false },
       resourceIndicators: {
         enabled: true,
+        useGrantedResource: () => false,
```

Comment:
should have some comments / link for this since it's for easier future recalling?

Reply:
good call